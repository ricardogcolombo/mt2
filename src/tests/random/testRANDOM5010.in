50 10
1 49 96 30 49
1 21 16 13 93
1 42 58 38 117
1 26 76 16 96
1 15 123 3 70
1 40 115 11 19
1 20 78 39 63
1 49 68 26 59
1 25 114 31 96
1 19 12 47 33
