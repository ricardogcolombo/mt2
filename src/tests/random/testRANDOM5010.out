0.500000
0.500000
0.375000
0.500000
0.500000
0.500000
0.500000
0.500000
0.500000
0.500000
0.375000
0.500000
0.625000
0.500000
0.625000
0.607143
0.500000
0.500000
0.375000
0.625000
0.375000
0.500000
0.500000
0.500000
0.625000
0.321429
0.500000
0.500000
0.500000
0.392857
0.375000
0.500000
0.500000
0.500000
0.500000
0.500000
0.500000
0.625000
0.375000
0.625000
0.500000
0.375000
0.500000
0.500000
0.500000
0.500000
0.625000
0.500000
0.678571
0.500000
